src/physics/systems.rs

```diff
@@ -60,12 +60,11 @@ pub fn create_joints_system(
 
 /// System responsible for performing one timestep of the physics world.
 pub fn step_world_system(
+    (time, mut sim_to_render_time): (Res<Time>, ResMut<SimulationToRenderTime>),
```

Comment:
Note that these variables had to be made into tuples of resources to work around the limit of 10 parameters to a Bevy system function.